include/fiction/algorithms/simulation/sidb/operational_domain.hpp

```diff
@@ -1168,6 +1334,32 @@ operational_domain operational_domain_contour_tracing(const Lyt& lyt, const std:
 
     return result;
 }
+/**
+ * This function computes the parameters necessary for ensuring the physical validity of the charge distribution and
+ * determines the corresponding excited state number. The ground state is denoted by zero, with each subsequent excited
+ * state incrementally numbered.
+ *
+ * @tparam Lyt The charge distribution surface type.
+ * @param cds The charge distribution surface for which physical parameters are to be determined.
+ * @param params Operational domain parameters.
+ * @return An instance of valid_physical_parameters_with_excited_state_number containing the determined physical
+ * parameters.
```

Comment:
```suggestion
 * @return An instance of `valid_physical_parameters_with_excited_state_number` containing the determined physical
 * parameters.
```
But it's not just the parameters.